frontend/src/core/sidebar/views/sidebarItemView.js

```diff
@@ -100,22 +100,19 @@ define(function(require) {
     },
 
     onKeyDown: function(e) {
-      window.clearTimeout(this.timeout);
-      this.timeout = -1;
-      switch(e.keyCode) {
-        case 17:
-        case 91:
-        case 93:
-          this.$('.editor-common-sidebar-preview')
-            .removeClass('action-primary')
-            .addClass('action-warning');
-          this.$('.editor-common-sidebar-preview-inner').text(window.polyglot.t('app.forcerebuild'));
-          this.forceRebuild = true;
-          this.timeout = window.setTimeout(_.bind(this.onKeyUp,this), 5000);
-          break;
-        default:
-          // do nothing
+      // 17: ctrl, 91: left cmd
+      if(e.keyCode !== 17 || e.keyCode !== 91) {
```

Comment:
Broken logic here – was this meant to be the below?
```js
if (!(e.keyCode === 17 || e.keyCode === 91))
```
I think the switch statement may read better.